rtsf-at-checkout-event-simulator/go.mod

```diff
@@ -1,8 +1,3 @@
 module event-simulator
```

Comment:
copyright

Reply:
we usually don't put copyright on go.mod file since it could be regenerated by go command. I checked go.mod from EdgeX edgex-go repo and it doesn't have copyright header either: https://github.com/edgexfoundry/edgex-go/blob/main/go.mod